### ISSUE TITLE
请问动态图PSPNet的mIoU是多少呢？

### ISSUE THREAD

**Issue by Yuuuuuuuuuuuuuuuuuummy** [PaddlePaddle/PaddleSeg#368]
关注到最近public了 PSPNet的动态图程序，想问一下复现的精度是多少？ 我实际运行这个repo里的代码精度总是上不去啊？

**Comment by michaelowenliu** [comment]
> 关注到最近public了 PSPNet的动态图程序，想问一下复现的精度是多少？ 我实际运行这个repo里的代码精度总是上不去啊？
Hi, Yumy, Thanks for your interest in the PSPNet version. In our early experiment, using OS=8, mIoU of pspnet_resnet50 is 79.6%, and pspnet_resnet101 is 79.84%.  Currently it is still working in progress with the api modifications, and the pertained model of backbone is not provided in the repo. Sorry for that.

**Comment by michaelowenliu** [comment]
@Yuuuuuuuuuuuuuuuuuummy 

Hi, Yueming, Thanks for your interest in the PSPNet version. In our early experiment, using OS=8, mIoU of pspnet_resnet50 is 79.6%, and pspnet_resnet101 is 79.84%. Currently it is still working in progress with the api modifications, and the pertained model of backbone is not provided in the repo. Sorry for that.

**Comment by Yuuuuuuuuuuuuuuuuuummy** [comment]
@michaelowenliu   Thx for the reply! 
Your experiments is implemented in cityscapes?
Have you implement the Sync_Batch_Norm in your experiment of paddle2.0? Sync_Batch_Norm is not supported in paddle1.8 dygraph version as far as I know. How much accuracy difference between sync_batch_norm and batch_norm do you think?

**Comment by michaelowenliu** [comment]
@Yuuuuuuuuuuuuuuuuuummy 
Hi, 

1. Yes, now we only conduct the experiment on Cityscapes, but VOC and ADK are on the way. 

2. Yes, SyncBn is really an important op. our Paddle team has already implemented SyncBn in paddle2.0 beta which will be released very soon in the middle of this month. In our experiment, we compiled paddle develop branch including SyncBn. 

3. As for accuracy difference, we did not do such comparison, since it depends on the practice (GPUs/batch size), but I guess there might be 1% difference.

btw, the accuracy of pretrained model of backbone also affects the results. The accuracy would be poor without pretrained model.

**Comment by Yuuuuuuuuuuuuuuuuuummy** [comment]
@michaelowenliu 
Sorry for reply late!
I'm working on re-producing PSPNet on paddle dygraph mode. Now I use paddle-1.8.3. I totally agree with you about the importance of pretrained model. I can't find a proper resnet101 dygraph model in Paddle. So I convert pytorch official pretrained model to paddle version. But there is still a little difference in performance.

Hope to get a stable dygraph version soon!
